src/case/search.rs

```diff
@@ -0,0 +1,61 @@
+pub(super) fn changes_when_casemapped_nonascii<const MAP_LOWER: bool>(
```

Comment:
there are two copies of this file (one in tabgen/src/search.rs) just to shut rust-analyzer up. its absolutely not needed.